docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,213 +1,134 @@
 ---
 title: Getting Started with Desktop Access
-description: Automatically add Teleport Desktop Access to your infrastructure
+description: Access Windows systems through Teleport Desktop Access
 ---
 
-<Details
-  title="Version warning"
-  opened={true}
-  scope={["oss", "enterprise"]}
-  scopeOnly={true}
-  min="10.2.6"
->
+## Prerequisites
 
-The features documented on this page are available in Teleport `10.2.6` and higher.
+- An existing Teleport cluster with one of the following versions:
 
-</Details>
+  **Open Source or Enterprise:** version 8.0 or newer
```

Comment:
For non-AD this will be 12.0 or newer.

Reply:
Yup, still need to update a lot in this. Draft PR.